ExtractUserDataActivity/handler.ts

```diff
@@ -463,27 +470,56 @@ export const queryAllUserData = (
       })
     ),
     // step 3: queries notifications statuses
-    TE.chain(
+    TE.bind("notificationStatuses", ({ notifications }) =>
+      findAllNotificationStatuses(notificationStatusModel, notifications)
+    ),
+    // step 4: queries profile service preferences
+    TE.bindW("servicesPreferences", () =>
+      pipe(
+        TE.tryCatch(
+          async () => servicePreferencesModel.findAllByFiscalCode(fiscalCode),
+          () =>
+            ActivityResultQueryFailure.encode({
+              kind: "QUERY_FAILURE",
+              reason: "Error while searching the profile service preferences"
+            })
+        ),
+        TE.map(flattenAsyncIterator),
+        TE.map(asyncIteratorToArray),
+        TE.chain(
+          (promise: Promise<ReadonlyArray<t.Validation<ServicePreference>>>) =>
+            TE.tryCatch(
+              () => promise,
+              () =>
+                ActivityResultQueryFailure.encode({
+                  kind: "QUERY_FAILURE",
+                  reason: "Error with the async operator"
+                })
+            )
+        ),
+        TE.map(ROA.rights)
```

Comment:
Add a comment in here to explain that we are filtering only right value obtained from the database (left represent malformed data stored inside the database)

Reply:
done in https://github.com/pagopa/io-functions-admin/pull/191/commits/b8fc8ca5c09962851f6e1e5838421523e7643955